### ISSUE TITLE
[BUG] The Mullvad GUI application is not usable and has graphical artifacts

### ISSUE THREAD

**Issue by milachew** [mullvad/mullvadvpn-app#5327]
## Issue report

Operating system: Fedora Workstation 38 (tested on Wayland and X.Org - same problem)

App version: 2023.5

### Issue description

It is not possible to use the GUI application.

When opened, the following window appears (see screenshot).

![Снимок экрана от 2023-10-19 12-02-40](https://github.com/mullvad/mullvadvpn-app/assets/113530911/1ee9259c-c2d7-4d60-a1ce-88b523cc2f54)

~~Clicking on the controls looks non-interactive, nothing changes when I try to click them.~~
_The elements still turned out to be interactive.
Opening the settings makes a completely branded blue color instead of a "porridge" of pixels._

At the same time, the VPN itself works and connects.
The CLI version is also normal.

If you need logs when trying to run from the terminal, please let me know how to do it, because there is a lot of text.
Basically, after the standard status text, there are a lot of lines of code in the log with errors of the kind:
```
Errors:
link failed but did not provide an info log
[28044:1019/121943.249453:ERROR:shared_context_state.cc(81)] Skia shader compilation error
```

**Comment by Vladdytine** [comment]
Same issue here on KALI after system upgrade

**Comment by milachew** [comment]
> Same issue here on KALI after system upgrade

I fixed it after reinstalling Mullvad... try this, please.
I will keep this issue open if you still have issue.

**Comment by Vladdytine** [comment]
> > Same issue here on KALI after system upgrade
> 
> I fixed it after reinstalling Mullvad... try this, please. I will keep this issue open if you still have issue.

Solved! I actually fixed it too, re-installing didn't help, but simply deleting the old cached files in Mullvad's cache-GPU folder in Home directory did the magic.

**Comment by raksooo** [comment]
This sounds like it could be related to https://github.com/mullvad/mullvadvpn-app/issues/4952. We've added automatic clearing of the cache on upgrade so hopefully this won't happen in the future, although it will become active when upgrading from 2023.6.
